gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java

```diff
@@ -309,22 +345,128 @@ protected HttpJsonOperationsStub(
+              .setRequestFormatter(
+                  ((ProtoMessageRequestFormatter<GetOperationRequest>)
+                          getOperationMethodDescriptor.getRequestFormatter())
+                      .toBuilder()
+                      .updateRawPath(customOperationHttpBindings.get(LRO_GET_OPERATION).getGet())
+                      .setAdditionalPaths(
+                          customOperationHttpBindings.get(LRO_GET_OPERATION)
+                              .getAdditionalBindingsList().stream()
+                              .map(HttpJsonOperationsStub::getValueBasedOnPatternCase)
+                              .collect(Collectors.toList()))
+                      .build())
+              .build();
+    }
+
+    if (customOperationHttpBindings.containsKey(LRO_DELETE_OPERATION)) {
+      deleteOperationMethodDescriptor =
+          deleteOperationMethodDescriptor
+              .toBuilder()
+              .setRequestFormatter(
+                  ((ProtoMessageRequestFormatter<DeleteOperationRequest>)
+                          deleteOperationMethodDescriptor.getRequestFormatter())
+                      .toBuilder()
+                      .updateRawPath(
+                          customOperationHttpBindings.get(LRO_DELETE_OPERATION).getDelete())
+                      .setAdditionalPaths(
+                          customOperationHttpBindings.get(LRO_DELETE_OPERATION)
+                              .getAdditionalBindingsList().stream()
+                              .map(HttpJsonOperationsStub::getValueBasedOnPatternCase)
+                              .collect(Collectors.toList()))
+                      .build())
+              .build();
+    }
+
+    if (customOperationHttpBindings.containsKey(LRO_CANCEL_OPERATION)) {
+      cancelOperationMethodDescriptor =
+          cancelOperationMethodDescriptor
+              .toBuilder()
+              .setRequestFormatter(
+                  ((ProtoMessageRequestFormatter<CancelOperationRequest>)
+                          cancelOperationMethodDescriptor.getRequestFormatter())
+                      .toBuilder()
+                      .updateRawPath(
+                          customOperationHttpBindings.get(LRO_CANCEL_OPERATION).getPost())
+                      .setAdditionalPaths(
+                          customOperationHttpBindings.get(LRO_CANCEL_OPERATION)
+                              .getAdditionalBindingsList().stream()
+                              .map(HttpJsonOperationsStub::getValueBasedOnPatternCase)
+                              .collect(Collectors.toList()))
+                      .build())
+              .build();
+    }
+  }
+
+  /**
+   * Wrapper to get a list of the MethodDescriptors similar to `getMethodDescriptors()`. Difference
+   * is that this gets the modified MethodDescriptors with any customized values
+   *
+   * @return List of the custom MethodDescriptors
+   */
+  @InternalApi
+  public List<ApiMethodDescriptor> getAllMethodDescriptors() {
```

Comment:
Not sure if I get the differences between this method and the one below, I think they should have the same behavior other other than the mutability of the list.

Reply:
I'll remove this one. 

It was intended for `static -> default method descriptors` and `non-static -> customized/ modified method descriptors`, but I think that just ends up making this confusing. I was worried that running these tests in parallel would mean that the method descriptors would be updated and different test cases would pulling in modified values, but the MethodDescriptors are static anyways it wouldn't resolve the issue.